bindings/rust/s2n-tls-tokio/src/lib.rs

```diff
@@ -203,6 +202,7 @@ where
             self.as_mut().set_receive_context(context)?;
             self.as_mut().set_send_context(context)?;
             self.as_mut().set_waker(Some(ctx.waker()))?;
+            self.as_mut().set_blinding(Blinding::SelfService)?;
 
```

Comment:
While updating my doc example I thought, "I'll use set_blinding instead of set_server_name" and then realized that s2n-tls-tokio probably did that automatically. I'm moving the call here so that the blinding can't be changed out from underneath s2n-tls-tokio.

I also checked for other connection-level configuration methods in s2n-tls-tokio and found none.